cmds/exp/ipmidump/ipmidump.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
```

Comment:
remove this. the copyright and the synopsis must be separated by at least one empty line, or the generated documents will include the copyright in the package description.